fluentlenium-core/src/main/java/org/fluentlenium/core/Fluent.java

```diff
@@ -360,6 +360,32 @@ public Fluent goTo(String url) {
         return this;
     }
 
+    /**
+     * Open the url page in a new tab
+     *
+     * @param url
+     */
+    public Fluent goToInNewTab(String url) {
```

Comment:
Is that method covered in unit tests? Adding new method requires to cover this by Unit tests in the test pkg.